RomanLoboda/HW11/Task2.py

```diff
@@ -0,0 +1,20 @@
+import logging
```

Comment:
good job
look at this version
import logging

logging.basicConfig(format='%(process)d-%(levelname)s-%(asctime)s-%(message)s', level=logging.INFO)

DAYS = ["monday", "thursday", "wednesday", "tuesday", "friday", "saturday", "sunday"]

def get_day_name():
    input_value = int(input("Enter a number: "))
    if input_value > 0 and input_value <= len(DAYS):
        return DAYS[input_value - 1]
    else:
        raise ValueError("Enter a positive number within the valid range.")

try:
    day_name = get_day_name()
    print(f"The day is {day_name}")
except ValueError as e:
    logging.warning(e)